axiom/nr_segment_datastore.c

```diff
@@ -50,28 +79,7 @@ static char* create_metrics(nr_segment_t* segment,
   nr_free(operation_metric);
   nr_free(statement_metric);
 
-  /*
-   * If a datastore instance was provided, we need to add the relevant data to
-   * the segment and the relavant metrics.
-   */
-  if (NULL == instance || 0 == txn->options.instance_reporting_enabled) {
-    return scoped_metric;
-  }
-
-  if (txn->options.database_name_reporting_enabled) {
-    nr_datastore_instance_set_database_name(&datastore->instance,
-                                            instance->database_name);
-  }
-
-  instance_metric = nr_formatf("Datastore/instance/%s/%s/%s", product,
-                               instance->host, instance->port_path_or_id);
-  nr_segment_add_metric(segment, instance_metric, false);
-  nr_datastore_instance_set_host(&datastore->instance, instance->host);
```

Comment:
These lines seem separate from the rest that was extracted into `create_instance_metric` and seems like it should have stayed in original function.

Reply:
refactored in [e2d7a78](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/e2d7a786bd73da46eef4fb0de581839e38e8e17f)